docs/developing/publishing-apps.md

```diff
@@ -0,0 +1,105 @@
+# App publishing guide
+
+After you've completed developing and testing your app's package for Sandstorm, you can publish it in the [Sandstorm App Market](https://apps.sandstorm.io/).
+
+## Verify your identity
+
+### Sign up with [Keybase.io](https://keybase.io)
+
+Currently, Keybase is invite-only. If you need an invite, you can contact [Jade](mailto:jade@sandstorm.io) or [Kenton](mailto:kenton@sandstorm.io). You should connect some of your public identites with your Keybase account, like Twitter and Github.
+
+Sandstorm app authors are verified using a PGP key linked to Keybase. You should get the [prerequisites](https://keybase.io/docs/command_line/prerequisites) and follow [their directions](https://keybase.io/docs/command_line/installation) to get their software set up.
+
+### Sign your app
+
+In order to verify that you are the author of the app in question, you need to sign the following ASCII statement: `I am the author of the Sandstorm.io app with the following ID: <app-id>`, where `<app-id>` is the one from your `sandstorm-pkgdef.capnp` file.
+
+To generate a pgp-signature file using gpg, run a command like this:
+
+`echo -n "I am the author of the Sandstorm.io app with the following ID: <app-id>" |
+     gpg --sign > pgp-signature`
```

Comment:
I think it’s worth including the following sentence here (from @dwrensha on IRC):

> If you do it correctly, `cat pgp-signature | gpg` should print out the statement that you signed.

For some reason `gpg --sign` didn't work for me, and I used `keybase sign` instead; but unfortunalty the resulting file didn't have the correct format. Doing this check earlier check would have saved me some time.